rhel/SPECS/pmm-server.spec

```diff
@@ -32,14 +32,16 @@ Requires(postun): systemd
 Percona Monitoring and Management (PMM) Server.
 See the PMM docs for more information.
 
-
 %prep
 %setup -q -n %{repo}-%{commit}
 sed -i "s/ENV_SERVER_USER/${SERVER_USER:-pmm}/g" prometheus.yml prometheus1.yml
 sed -i "s/ENV_SERVER_PASSWORD/${SERVER_PASSWORD:-pmm}/g" prometheus.yml prometheus1.yml
 echo "${SERVER_USER:-pmm}:$(openssl passwd -apr1 ${SERVER_PASSWORD:-pmm})" > .htpasswd
 sed -i "s/v[0-9].[0-9].[0-9]/v%{version}/" landing-page/index.html
 
+%build
+make build-password-page
+
```

Comment:
also good practice to have 2 empty lines between sections

Reply:
Done